### ISSUE TITLE
Congratulations and Questions

### ISSUE THREAD

**Issue by BraudoCC** [TheDatumOrg/TSB-AD#43]
Congratulations on achieving such great results! I encountered a question during my study: 

I found that the test set of the experiment contains data from the training set. Is this kind of setup widely applied? Because I noticed that if I change the test set to the last 30% of the data, there are no anomaly samples in some time-series data. 

Also, have you considered open-sourcing the original data?

**Comment by qhliu26** [comment]
Hi, thank you for your interest in our work! Please refer to the following discussion for details on the training/test data splitting:
https://github.com/TheDatumOrg/TSB-AD/issues/6#issuecomment-2550137783.

The original dataset can be accessed here: https://www.thedatum.org/datasets/TSB-UAD-Public-v2.zip
(It contains 29 datasets and 3,427 time series.)

Let us know if you have any further questions!

**Comment by qhliu26** [comment]
Marking this issue as fixed and closing.
